planet/clients/subscriptions.py

```diff
@@ -87,10 +126,12 @@ async def create_subscription(self, request: dict) -> dict:
             # TODO: replace with httpx request.
             sub = await _server_subscriptions_post(request)
         except Exception as server_error:
-            # TODO: remove "from server_error" clause.
+            # TODO: remove "from server_error" clause. It's useful
+            # during development but may invite users to depend on the
+            # value of server_error.
             raise ClientError("Subscription failure") from server_error
-
-        return sub
+        else:
+            return sub
```

Comment:
`sub` only being defined in the `try` clause, it seems good to keep the reference under an `else`.